README.md

```diff
@@ -431,7 +431,9 @@ A number of options are available to allow you to customise the SDK:
 
   - `showCountrySelection` (boolean - default: `false`)
 
-    If only one document type is preselected that is not a passport document, the issuing country selection screen can still be displayed by setting this to `true`.
+    The `showCountrySelection` option controls what happens when **only a single document** is preselected in `documentTypes` It has no effect when the SDK has been set up with multiple documents preselected.
+    The country selection screen is never displayed for a passport document.
```

Comment:
line breaks are missing when I am viewing the output, can we force the linebreaks to make the text easier to read?

Reply:
Unfortunately this seems to be how the line breaks are spaced on Github despite additional white space between lines.